app/models/user.rb

```diff
@@ -9,6 +9,9 @@ class User < ApplicationRecord
 
 	validates :first_name, :last_name, :city_id,  presence: true
 
+  # enums
+  enum role: [:citizen, :admin]
```

Comment:
@mikevic  - Can we start defining the enum like {citizen: 0, admin: 1}. In case if we want to remove something it will be easier.

Reply:
Agreed, will do the change in #19

Reply:
cool bro.